Xamarin.Forms.Core/Span.cs

```diff
@@ -1,5 +1,9 @@
 using System;
+using Xamarin.Forms;
 using Xamarin.Forms.Internals;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-color", typeof(Span), nameof(Span.BackgroundColorProperty))]
```

Comment:
`color` is already mapped to TextColorProperty, and `background-color` to `BackgroundColorProperty`